tonic-build/src/client.rs

```diff
@@ -137,6 +137,20 @@ pub(crate) fn generate_internal<T: Service>(
                     self
                 }
 
+                /// Limits the maximum size of a decoded message.
+                #[must_use]
+                pub fn max_decoding_message_size(mut self, limit: usize) -> Self {
+                    self.inner = self.inner.max_decoding_message_size = Some(limit);
```

Comment:
I don't think this is valid rust xD

Reply:
I just copy and past from the code above 

```rust
                /// Compress requests with the given encoding.
                ///
                /// This requires the server to support it otherwise it might respond with an
                /// error.
                #[must_use]
                pub fn send_compressed(mut self, encoding: CompressionEncoding) -> Self {
                    self.inner = self.inner.send_compressed(encoding);
                    self
                }

                /// Enable decompressing responses.
                #[must_use]
                pub fn accept_compressed(mut self, encoding: CompressionEncoding) -> Self {
                    self.inner = self.inner.accept_compressed(encoding);
                    self
                }
```